pybamm/parameters/thermal_parameters.py

```diff
@@ -0,0 +1,72 @@
+#
+# Standard thermal parameters
+#
+import pybamm
+
+# --------------------------------------------------------------------------------------
+# Dimensional parameters
+
+# Reference temperature
+T_ref = pybamm.Parameter("Reference temperature [K]")
+
+# Density
+rho_cn_dim = pybamm.Parameter("Negative current collector density [kg.m-3]")
+rho_n_dim = pybamm.Parameter("Negative electrode density [kg.m-3]")
+rho_s_dim = pybamm.Parameter("Separator density [kg.m-3]")
+rho_p_dim = pybamm.Parameter("Positive electrode density [kg.m-3]")
+rho_cp_dim = pybamm.Parameter("Positive current collector density [kg.m-3]")
+
+# Specific heat capacity
+c_p_cn_dim = pybamm.Parameter(
+    "Negative current collector specific heat capacity [J.kg-1.K-1]"
+)
+c_p_n_dim = pybamm.Parameter("Negative electrode specific heat capacity [J.kg-1.K-1]")
+c_p_s_dim = pybamm.Parameter("Separator specific heat capacity [J.kg-1.K-1]")
+c_p_p_dim = pybamm.Parameter("Negative electrode specific heat capacity [J.kg-1.K-1]")
+c_p_cp_dim = pybamm.Parameter(
+    "Positive current collector specific heat capacity [J.kg-1.K-1]"
+)
+
+# Thermal conductivity
+lambda_cn_dim = pybamm.Parameter(
+    "Negative current collector thermal conductivity [W.m-1.K-1]"
+)
+lambda_n_dim = pybamm.Parameter("Negative electrode thermal conductivity [W.m-1.K-1]")
+lambda_s_dim = pybamm.Parameter("Separator thermal conductivity [W.m-1.K-1]")
+lambda_p_dim = pybamm.Parameter("Positive electrode thermal conductivity [W.m-1.K-1]")
+lambda_cp_dim = pybamm.Parameter(
+    "Positive current collector thermal conductivity [W.m-1.K-1]"
+)
+
+# Thermal parameters
+h_dim = pybamm.Parameter("Heat transfer coefficient [W.m-2.K-1]")
+Delta_T = pybamm.Parameter("Typical temperature variation [K]")
```

Comment:
To get the scaling for Delta T we say it scales like IV/h, so could define it here as `Delta_T = ((U_p_ref - U_n_ref) + (R*T_ref/F)) *(I_typ/h)` to be more transparent about where it comes from. I think originally we had `Delta_t = I_typ*Phi_ref / h_ref`, with phi_ref = 1V, but should give the same order of magnitude